thoth/storages/graph/postgres.py

```diff
@@ -2317,17 +2316,11 @@ def get_python_package_version_dependents_all(
                         "marker_evaluation_result": entry[4],
                         "marker": entry[5],
                         "extra": entry[6],
-                        "platform": entry[7],
```

Comment:
Not sure which components use this query actually

Reply:
revsolver uses this query
https://github.com/thoth-station/revsolver/blob/e928e373428c909638339198c7369a134616e851/app.py#L64

Reply:
thanks! But the platform itself is not required, so removing it, won't break it in theory. ack?

Reply:
yes, correct it wont, break it.